src/components/NavigationBar/style.scss

```diff
@@ -113,9 +113,9 @@
 }
 // Styling for menu based nav bar
 
-@media only screen and (min-width: 900px) {
+@media only screen and (min-width: 900.1px) {
```

Comment:
Why 900.1?

Reply:
i was just being really nitpicky. if i set the breakpoint at 900px, it breaks at 900px and if i set it at 901px, it breaks between 900px and 901px (which you can achieve if you zoom in on the page). i wish media queries allowed you to use operators like < vs <= to avoid overlapping selectors but thats not a thing :(

Reply:
nvm ignore that code i found a much easier way to fix it